src/ansible_runner/utils/__init__.py

```diff
@@ -386,14 +386,14 @@ def _emit_event(self,
             event_data = self._current_event_data
             stdout_chunks = [buffered_stdout]
         elif buffered_stdout:
-            event_data = dict({'event': 'verbose'})
+            event_data = dict({'event': 'unclassified_display'})
```

Comment:
See https://github.com/ansible/ansible-runner/pull/1345/files#r1523986425

Reply:
All of the CI failures appear to be related to this, where `verbose` is being excluded, or depended on. I'm not sure we can make this change, but I'd prefer to not just classify any unknown display event as verbose if possible.